### DISCUSSION TITLE
Is there a way to replicate Fooocus output in Forge UI/A1111 ?

### DISCUSSION THREAD

**Discussion by Uneternal** [lllyasviel/Fooocus#2310] [General]
Even when I switch off any optimizations (any styles like V2 + the offset lora) the pictures come out completely different from what I get in Forge UI. Is there a way to somehow get the same output?

![Screenshot 2024-02-20 101410](https://github.com/lllyasviel/Fooocus/assets/32781719/f3400068-9e66-4c3a-a5e4-2e8182c75c2f)
![Screenshot 2024-02-20 103015](https://github.com/lllyasviel/Fooocus/assets/32781719/33ea592e-eaaf-4f1e-bd4d-3eb26415d01d)

**Comment by mashb1t** [comment]
There currently is no such possibility, as they use different backends afaik. Forge uses A1111 and Fooocus uses ldm_patched.

**Reply by R4290N** [reply]
please help me.
I used Focus, overlaying SD1.5 as a refiner on SDXL models.
How can I repeat this process in Forge?
All I could think of is to put on SDXL model generation 12 steps and enable HiResFix with scale factor =1 and including there SD1.5 model with denoising about 0.75.
The result is somewhat similar to what I want, but I'm still not very satisfied.
please give me your advice.

**Reply by mashb1t** [reply]
you can enable refiner also in txt2img and choose a SD1.5 model, no hires fix needed

**Comment by barepixels** [comment]
someone else said "Different implementations of scheduler , latent, etc. No possible to get same results."

**Comment by MohammadAboulEla** [comment]
could you help me understand hat is ldm_patched?

**Reply by mashb1t** [reply]
https://github.com/lllyasviel/Fooocus/tree/main/ldm_patched is the library of Fooocus used for the Stable Diffusion image generation pipeline, model management, resource optimizations etc.
